css/style.css

```diff
@@ -1,139 +1,338 @@
-  }
-  section{
-    width: 950px;
-    margin: 0 auto;
-  }
-p {
-  padding-top: 20px;
-  padding-bottom: 20px;
-}
-
-
-img {
-  max-width: 100%;
-  max-height: 100%;
+
+.center {
+  position: absolute;
+  top:0;
+  left:40%;
+  font-family: 'Pattaya', sans-serif;
+}
+
+.left{
+  position: absolute;
+  left:50px;
+  top:0;
+  
+ 
+}
+.left h3 {
+  display:inline;
+}
+
+
+.details{
+  width:600px;
+  margin: auto;
+  text-align: justify;
+  margin-bottom:60px;
+ 
+}
+
+.details_img {
+  width:1200px;
   margin: auto;
+  text-align: justify;
+  margin-bottom:60px;
+}
+
+.details_img h2 {
+  font-style: italic;
+  margin: 2rem auto;
+  padding-top:  20px;
+  margin-left: 200px;
+}
+
+.details .date{  
+  float:left;
+ 
+}
+.details .icons{
+
+float:right;
+
+}
+
+.details p{
+margin-top:40px;
+font-size: 20px;
+line-height: 30px;
+margin-bottom:20px;
+}
+
+.two_sides{
+position: relative;
+}
+
+
+.left_side{
+  text-align: justify;
+  font-size: 20px;
+  line-height: 30px;
+  max-width:800px;
+  
   
 }
-.wrapper{
-  position: relative;
+.left_side p{
+  margin-top:40px;
+  margin-bottom:20px;
+}
+
+.right_side{
+max-width:220px;
+margin-top: 160px;
+position: absolute;
+right:10vw;
+top:60px;
+}
+.img-div {
+ display:inline-block;
+}
+
+.img-div img{
+ float:left;
+ margin:5px;
+  
+}
+
+.img-div p{
+  padding-top:6px;
+}
+
+.signup{
+  width:200px;
+  line-height:15px;
+}
+
+.signup button{
+  width:200px;
+  box-sizing: border-box;
+  padding:5px 20px;
+  margin: 10px 0;
+}
+.signup h3 {
+  margin: 50px 0 10px 0;
+}
+.overflow_hide{
+  overflow:hidden;
+  margin:auto;
+}
+.overflow_hide h3 {
+  margin-bottom: -30px;
+}
+.inflation_text p{
+  font-size:15px;
+  line-height:20px;
+  text-align: left;
+}
+
+.theory_items {
+  width: 700px;
+  background-image: url(https://static01.nyt.com/newsgraphics/2014/03/18/theory-of-inflation/1ebaa1b972edda0f407fbaa0e010f6b50fb6792e/inflation-720.png);
+  background-position:left;
+  background-size:700px 850px;
+  background-repeat: no-repeat;
+ 
+  display:grid;
+  grid-template-columns: 25% 25% 25% 25%;
+  grid-template-rows: 50% 50%;
+  grid-gap: 1% 1%;
 }
-.container {
-  height: 90vh;
-  margin: 2rem;
+
+.item {
+  width:170px;
+  margin-top:100px;
+}
+
+#cosmos{
+  display:flex;
+  width:90%;
+  justify-content: center;
+}
+.title {
+  margin-top: -50px;
+}
+.title hr {
+  margin-bottom:10px;
+  width:50px;
+}
+
+.title p{
+  font-size:20px;
+  line-height: 20px;
+  margin-top:20px;
+}
+
+
+.figcap {
+  font-size:12px !important;
+  color:gray !important;
+  line-height:15px !important;
+  margin-bottom:20px !important;
+  text-align:center !important;
+}
+
+
+
+.read {
   display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr; 
-  grid-gap: 20px;
-  position: absolute;
-  top: 90px;
-  left 0;
+  grid-template-columns: 50% 50%;
+  grid-column-gap: 19vw;
+  
+}
+
+.details_5{
+  height:300px;
+  width:80%;
+  margin:40px auto;
+  display: flex;
+  justify-content: center;
+}
+
```

Comment:
It seems like you are not using this class in the HTML so, in that case, it's better to delete it.